### ISSUE TITLE
Bodymovin with Natron

### ISSUE THREAD

**Issue by koder12** [NatronGitHub/Natron#283]
Can we use Bodymovin with Natron for complex SVG animations?

**Comment by devernay** [comment]
I don't know about bodymovin, but there are people who are doing SVG animation with Natron:
- https://discuss.pixls.us/t/natron-for-2d-animation/7779
- https://colorgrade13.wordpress.com/2016/06/26/natron-inkscape-svg-workflows/ 
- http://natron.readthedocs.io/en/rb-2.3/guide/tutorials-svgworkflow.html

Do not forget to render your SVG at a higher DPI if you intend to transform it in the compositing graph.

**Comment by grrinch** [comment]
👍 +1
This would be absolutely awesome to have in Natron!

**Comment by rodlie** [comment]
Natron is probably the wrong tool for this job. Natron is a raster application, you can't work or export in vector.

**Comment by m1ga** [comment]
That's true. Synfig got a GSoC PR with Lottie export: https://github.com/synfig/synfig/pull/756 so that would be the best OSS/Linux solution for vector based animations with Lottie export

**Comment by mikwee** [comment]
**Exporting** to Lottie may not be of much use, but you know what could be incredibly useful? **Importing** from Lottie. Natron isn't the best at motion graphics, so maybe doing specific animations on a dedicated animation software (AE, Synfig, Haiku Animator etc.) could ease the burden a bit.
